queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -177,6 +177,28 @@ def _load_reduction_script(self, instrument_name):
         """ Returns the path of the reduction script for an instrument. """
         return os.path.join(self._reduction_script_location(instrument_name), 'reduce.py')
 
+
+    def specify_instrument_directories(self,
+                                       instrument_output_directory,
+                                       excitations,
+                                       temporary_directory):
+        """
+        Specifies instrument directories, including removal of run_number folder
+        if excitations instrument
+        :param instrument_output_directory: (str) Ceph directory using instrument, proposal, run no
+        :param excitations: (list) List of excitations instruments
+        :param temporary_directory: (str) Temp directory location (root)
+        :return (str) Directories where Autoreduction should output
+        """
+
+        if self.instrument in excitations:
+            # Excitations would like to remove the run number folder at the end
+            no_run_number_directory = instrument_output_directory.rfind('/') + 1
```

Comment:
Were still not asserting or checking that `instrument_output_directory` has the right format

If someone changed the call from `ALF/RB123/50005` to just `ALF/RB123` (such as a bug) we would just dump data into `ALF/`causing someone a painful cleanup down the road rather than throwing an assert error